src/net.rs

```diff
@@ -230,7 +270,10 @@ pub fn test_read_commands(){
     // read the command from the stream for Command::Quit
     let mut command_res = read_commands(&mut Cursor::new(vec));
     assert_eq!(command_res.is_ok(), true);
-    assert_eq!(command_res.unwrap(), Command::Quit);
+    match command_res.unwrap() {
+        Command::Quit => assert!(true),
+        _ => assert!(false)
+    };
```

Comment:
I've tried but it does not work. It's not the struct Command that causes the problem, the io::Error, bincode::Error etc. that do not implement PartialEq.